tests/providers/amazon/aws/operators/test_emr_add_steps.py

```diff
@@ -224,4 +224,5 @@ def test_wait_for_completion(self, mock_add_job_flow_steps):
             job_flow_id=job_flow_id,
             steps=[],
             wait_for_completion=False,
+            execution_role_arn=None,
         )
```

Comment:
Could we also have a test scenario in Hook?
1. Provide value
2. Provide empty value: `None`, `""`